docs/source/performance/caching.md

```diff
@@ -17,25 +19,25 @@ type Post {
 }
 ```
 
- When Apollo Server resolves an operation, it calculates the result's correct cache behavior based on the _most restrictive_ settings among the result's fields. You can then use this calculation to support any form of cache implementation you want, such as by providing it to your CDN via a `Cache-Control` header.
+When Apollo Server resolves an operation, it calculates the result's correct cache behavior based on the [_most restrictive_ settings](#calculating-cache-behavior) among the result's fields. You can then use this calculation to support any form of cache implementation you want, such as by providing it to your CDN via a `Cache-Control` header.
 
 You can define field-level cache settings [statically in your schema definition](#in-your-schema-static) (as shown above) or [dynamically in your resolvers](#in-your-resolvers-dynamic).
 
-## Field settings
+## Setting cache hints
 
 When caching operation results, it's important to understand:
 
 * Which fields of your schema can be cached safely
 * How long a cached value should remain valid
 * Whether a cached value is global or user-specific
 
-These details can vary significantly, even among the fields of a single object type. You can specify these details statically in your schema definition or dynamically in your resolvers.
```

Comment:
Hmm, I thought this sentence was helpful, as part of the "introductory" part of this `##` section. Maybe it should be its own paragraph?

Reply:
Cool, this sentence was effectively a duplicate of one just above, but I'll delete _that_ one and keep the one here instead.

Reply:
Oh I guess that isn't too far away. Yeah, leave in whichever one seems best.